src/main/resources/default/templates/biz/storage/list.html.pasta

```diff
@@ -46,22 +46,22 @@
                     <t:filterbox page="children" baseUrl="@apply('/fs?path=%s', dir.path())"/>
                 </i:block>
 
-                <div class="row">
-                    <t:datacard>
-                        <table class="table table-hover">
+                <div class="card mb-4">
+                    <div class="card-body">
+                        <table class="table table-hover" style="table-layout: fixed">
                             <thead>
                             <tr>
-                                <th>
+                                <th scope="col">
                                     @i18n("Model.name")<br>
-                                    <small class="text-muted">@i18n("Model.description")</small>
+                                    <small class="text-muted text-nowrap">@i18n("Model.description")</small>
```

Comment:
Does `text-nowrap` work or do we actually need `text-truncate` here?

Reply:
text-nowrap is ok here, only to now break "Description"/"Beschreibung" for the last letter on really small screens